src/SparceCoverageCollector.js

```diff
@@ -0,0 +1,119 @@
+class SparceCoverageCollector {
```

Comment:
We're not generally big fans of ES6 classes because they don't add much and don't provide the safeguards you would expect from classes. That said, that's probably fine here.